plugins/transport-grpc/src/main/proto/opensearch/main_action.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
```

Comment:
imo, we should not have ... such handwrite proto file in core repo, esp, it is the client <> server api. (though it's empty api). can we keep the PR focusing on grpc server.

Reply:
+1 would prefer not to have to review API implementations and their integration tests in the same diff as the GRPC server

Reply:
:thumbsup: moved to just default health/reflection services provided by io.grpc package.